Gemfile

```
@@ -31,6 +31,7 @@ gem "bundler",                          "~> 2.1", ">= 2.1.4", "!= 2.2.10", "< 2.
 gem "byebug",                                                :require => false
 gem "color",                            "~>1.8"
 gem "config",                           "~>2.2", ">=2.2.3",  :require => false
+gem "cgi",                              "~> 0.1.0.2"                           # CVE-2021-33621 fixed: ruby 2.7.7 - https://github.com/advisories/GHSA-vc47-6rqg-c7f5
```

Should this have require false?

```suggestion
gem "cgi",                              "~> 0.1.0.2", :require => false        # CVE-2021-33621 fixed: ruby 2.7.7 - https://github.com/advisories/GHSA-vc47-6rqg-c7f5
```

---

Similarly to uri below, are we locking to 0.1.0, or should this be `"~> 0.1.0", ">= 0.1.0.2"`?

---

updated.

TBH/ probably `~> 0.1.0` would have been fine (we'd end up with 0.1.1)